homeassistant/components/teslemetry/strings.json

```diff
@@ -274,5 +286,10 @@
         "name": "State code"
       }
     }
+  },
+  "exceptions": {
+    "no_cable": {
+      "message": "No charge cable connected"
```

Comment:
Should this message be clearer that locking is automatic connecting the cable? Not entirely clear this that you can't manually lock.

Reply:
The idea is that you can lock a cable that isnt attached, but ill try add something to it.